components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountArabicSnapshotTests.m

```diff
@@ -41,11 +41,6 @@ - (void)setUp {
       [[MDCTextInputControllerOutlinedTextArea alloc] initWithTextInput:self.textField];
   self.textFieldController.characterCountMax = 50;
   self.textFieldController.characterCountViewMode = UITextFieldViewModeAlways;
-  MDCSemanticColorScheme *colorScheme =
```

Comment:
On line 15, can you remove the ColorThemer imports? I would also expect the BUILD/Podspec file to be updated with the removed dependencies for the examples.

Reply:
Removed imports and BUILD deps 👍